google/cloud/pubsub/BUILD.bazel

```diff
@@ -38,6 +38,8 @@ cc_library(
     deps = [
         "//:common",
         "//:grpc_utils",
+        "@com_google_googleapis//google/cloud/location:location_cc_grpc",
```

Comment:
necessary? It does not look like pubsub is mixing in the locations service.

Reply:
Also how do you plan to update these `BUILD` files for other services? Manually?

In CMake, the equivalent of `//:grpc_utils` is depending on the mixin services. I am not sure Bazel's `layering_check` will allow that though.

Reply:
I should remove `location_cc_grpc`. I planned to add the dependencies for each library. Do you know where I can add the mixin dependencies for bazel like what I did for cmake in `google/cloud/google_cloud_cpp_grpc_utils.cmake`? That will save me a lot of efforts.

Reply:
https://github.com/googleapis/google-cloud-cpp/blob/3601f912caad97b242059ac1af336becc71d659e/google/cloud/BUILD.bazel#L166-L176

Hopefully bazel lets us pick up these dependencies transitively... :crossed_fingers:

Reply:
Great! It works.